packs/equipment-effects/effect-focus-on-the-doomed-fear-penalty.json

```diff
@@ -0,0 +1,86 @@
+{
+    "_id": "ofEoHZzaVmpKrMVH",
+    "img": "icons/equipment/wrist/bracer-red-silver.webp",
+    "name": "Effect: Focus on the Doomed (Fear Penalty)",
+    "system": {
+        "description": {
+            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Assassin's Bracers (Type I)], @UUID[Compendium.pf2e.equipment-srd.Item.Assassin's Bracers (Type II)], @UUID[Compendium.pf2e.equipment-srd.Item.Assassin's Bracers (Type III)]</p>\n<p>You take an item penalty to saving throws against fear effects.</p>"
+        },
+        "duration": {
+            "expiry": null,
+            "sustained": false,
+            "unit": "unlimited",
+            "value": -1
+        },
+        "level": {
+            "value": 10
+        },
+        "publication": {
+            "license": "ORC",
+            "remaster": true,
+            "title": "Pathfinder Adventure: Prey for Death"
+        },
+        "rules": [
+            {
+                "adjustName": false,
+                "choices": [
+                    {
+                        "label": "PF2E.SpecificRule.BonusLabel.NegativeOne",
+                        "value": -1
+                    },
+                    {
+                        "label": "PF2E.SpecificRule.BonusLabel.NegativeTwo",
+                        "value": -2
+                    },
+                    {
+                        "label": "PF2E.SpecificRule.BonusLabel.NegativeThree",
+                        "value": -3
+                    }
+                ],
+                "flag": "penalty",
+                "key": "ChoiceSet",
+                "predicate": [
+                    {
+                        "not": "parent:origin:item"
+                    }
+                ],
+                "prompt": "PF2E.SpecificRule.Prompt.Penalty"
+            },
+            {
+                "key": "FlatModifier",
+                "predicate": [
+                    "item:trait:fear",
+                    {
+                        "not": "parent:origin:item"
+                    }
+                ],
+                "selector": "saving-throw",
+                "slug": "focus-on-the-doomed-no-origin",
+                "type": "item",
+                "value": "@item.flags.pf2e.rulesSelections.penalty"
```

Comment:
Unfortunately FlatModifier currently emits a warning when a value cannot be resolved, even if the predicate doesn't pass. However, this doesn't affect the functionality.

The warning differs depending on how it was applied, and thus which value cannot be resolved:

`PF2e System | Flat Modifier rules element on item Effect: Focus on the Doomed (Fear Penalty) (Actor.rTiwxgxYljrE34aJ.Item.HXJaEh3LGmdkTSu1) failed to validate: unable to resolve formula, "-@item.origin.flags.pf2e.assassinsBracersBonus"`
`PF2e System | Flat Modifier rules element on item Effect: Focus on the Doomed (Fear Penalty) (Actor.rTiwxgxYljrE34aJ.Item.Y8qYnnQVwlkaAJIg) failed to validate: unable to resolve formula, "@item.flags.pf2e.rulesSelections.penalty"`

Reply:
We can dance around this by making a single FlatModifier with a static value and then using AdjustModifier. Cleaner in the roll dialog as well